app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The Person has not been vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("The Person have outdated vaccine")
+        elif ("wearing_a_mask" not in visitor
+              or visitor.get("wearing_a_mask") is False):
+            raise NotWearingMaskError("The Person without mask")
```

Comment:
This is a code style issue. According to the checklist, you should use descriptive error messages. Here, the error message is not descriptive enough. You should specify why the person is not allowed to enter. For example: 'The person is not wearing a mask.'.